src/pages/IndividualFundraiserPage.js

```diff
@@ -1,26 +1,69 @@
 import React, {useState, useEffect} from 'react'
 import {  useParams } from "react-router-dom";
-import placeholderImage from '../imgs/Fundraising-Tree.jpg'
+import placeholderImage from '../imgs/placeHolderFundraiserPic.jpg'
 
 const IndividualFundraiserPage = () => {
-    // const [fundraiserRequestOptions, setFundraiserRequestOptions] = useState({});
     const [fundraiser, setFundraiser] = useState([]);
+    const [user, setUser] = useState([]);
     const [tags, setTags] = useState([]);
-    const {fundraiserId} = useParams();
-
-    // Set the request url
-    const url = "https://localhost:7000/api/fundraiser/".concat(fundraiserId);
+    const {userId, fundraiserId} = useParams();
+    
+    // Set the fundraiser and user urls
+    const userUrl = "https://localhost:7000/api/user/".concat(userId);
+    const fundraiserUrl = "https://localhost:7000/api/fundraiser/".concat(fundraiserId);
 
-    // Number formatter function for cuirrency
+    // Number formatter function for currency
     const currencyFormatter = new Intl.NumberFormat('en-US', {
         style: 'currency',
         currency: 'USD',
-    
+        
         // These options are needed to round to whole numbers if that's what you want.
         minimumFractionDigits: 0, // (this suffices for whole numbers, but will print 2500.10 as $2,500.1)
         maximumFractionDigits: 0, // (causes 2500.99 to be printed as $2,501)
     });
 
+    // Formats the date from database format to MM/dd/yyyy
+    const formatDate = (date) => {
+        return (
+            date?.split("T")[0].split("-")[1] 
+            + "-" + date?.split("T")[0].split("-")[2] 
+            + "-" + date?.split("T")[0].split("-")[0]
+            )
+    };
```

Comment:
To make things easier you can import:

import { format } from 'date-fns'

and format the date with:

{format(new Date(props.createdDate), 'MM/dd/yyyy')}

Reply:
Im getting this error when I try to use this format function: 
<img width="637" alt="image" src="https://user-images.githubusercontent.com/77643903/222871374-c8ddd2ca-a0b7-462f-addf-6c06ea5ae4c2.png">

Reply:
actually I got it to work by doing a null check before calling the function

Reply:
Riiight I know why its like that. Actually when I pass the data into your fundraiser component you dont even have to worry about formatting because I'll handle it from the fetch. You can just leave it for now, its good